### PR TITLE
Add more info of SHUI

### PR DESCRIPTION
**Pull request by larkloss**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
SHUI

##### Twitter Link: 
https://twitter.com/shui_finance

##### List of audit links if any: 
https://cert-api.salusec.io/api/v1/salus/contract/certificate/full/Shui-LST_audit_report_2024-04-10.pdf

##### Website Link: 
https://app.shui.finance/

##### Logo (High resolution, will be shown with rounded borders):
https://drive.google.com/file/d/1Sf1JHHx1nuMJwIXFeyBGMiY87LrXOqze/view?usp=sharing

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
https://www.coingecko.com/en/coins/shui

##### Short Description (to be shown on DefiLlama):
SHUI is a decentralized non-custodial liquid staking protocol on Conflux. SHUI allows users to earn rewards securely and smoothly on their assets without locking the assets up.

##### Token address and ticker if any: 
Token ticker: SHUI
Token address: 0xf1f6e3aa98bac6c13230051e452065df299a78a7

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Liquid Staking

##### Github org/user (Optional, if your code is open source, we can track activity): 
https://github.com/Shui-LST

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/shui exports TVL: 
        
 
 
    conflux                   3.69 M
    conflux-pool2             0
    pool2                     0
    
    total                    3.69 M

File: projects/shui/index.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 const ADDRESSES = require('../helper/coreAssets.json')
 const SCFX_TokenAddress = "0x1858a8d367e69cd9E23d0Da4169885a47F05f1bE";
+const { sumTokensExport, } = require("../helper/unknownTokens");
+const MasterchefV2 = "0xeced26633b5c2d7124b5eae794c9c32a8b8e7df2";
```
Comment (g1nt0ki) [inline projects/shui/index.js:4]: is this masterchef deployed/maintained by your team?

Comment (larkloss) [reply projects/shui/index.js:4]: No, it's made by nucleon I guess, we contact them,they said we could use it to calculate

Comment (g1nt0ki) [reply projects/shui/index.js:4]: sorry, we cant count something as pool2 if the contract is not owned by your team.

Comment (larkloss) [reply projects/shui/index.js:4]: OK, that's fine. Do I need to lanuch a new PR or just delete the line?

Comment (g1nt0ki) [reply projects/shui/index.js:4]: there is no change in the adapter except for that right? what info do you want to update here? https://defillama.com/protocol/shui#information

**Comment by llamatester** [comment]
The adapter at projects/shui exports TVL: 
        
 
 
    conflux                   3.75 M
    conflux-pool2             0
    pool2                     0
    
    total                    3.75 M

**Comment by llamatester** [comment]
The adapter at projects/shui exports TVL: 
        
 
 
    conflux                   3.76 M
    conflux-pool2             465.53 k
    pool2                     465.53 k
    
    total                    3.76 M

**Comment by g1nt0ki** [comment]
thanks for the update

**Pull request merged**